### ISSUE TITLE
Domain and request table: search button screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2907]
### Issue description

The search buttons on the page do not have sufficient details within their accessible names to distinguish them from one another. This means screen reader users we'll hear the same label for different buttons, making it difficult to understand what each button is associated with. Each button should have a unique and descriptive accessible name that clearly indicates its purpose and content it relates to. This will help screen reader users to easily add testify and interact with the buttons.

### Acceptance criteria

- [ ] The domain table and request table search input field has the screen reader output: Search by domain name
- [ ] The domain table and request table  button has the screen reader output: Search by domain name
